src/docker/build/docker-image/Dockerfile.alpine

```
@@ -0,0 +1,111 @@
+# Stage 3: Final Runtime Image
+# ==============================================================================
+FROM alpine:3.21 AS runtime
+
+LABEL maintainer="nitrobass24" \
+      description="SeedSync - Seedbox file synchronization tool"
+
+# Copy stripped Python runtime from python-deps stage
+COPY --from=python-deps /usr/local/ /usr/local/
+
+# Copy unrar binary from linuxserver (not packaged in Alpine)
+COPY --from=linuxserver/unrar /usr/bin/unrar-alpine /usr/bin/unrar
```

Hadolint (2.14.0)

[warning] `COPY --from` should reference a previously defined `FROM` alias

(DL3022)